test_scripts/Polices/External_UCS/External_Consent_OFF/030_ATF_Policies_External_Consent_OFF_empty_consent_groups.lua

```diff
@@ -0,0 +1,173 @@
+--------------------------------------Requirement summary---------------------------------------------
+-- [Policies] External UCS: empty "consent_groups" changes after externalConsentStatus was received
+
+------------------------------------General Settings for Configuration--------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/ATF_Policies_External_Consent_common_functions')
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
+
+---------------------------------------Common Variables-----------------------------------------------
+local policy_file = config.pathToSDL .. "storage/policy.sqlite"
+
+---------------------------------------Preconditions--------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
+
+------------------------------------------Tests-------------------------------------------------------
+-- TEST 07:
+-- In case
+-- SDL Policies database does not have "consent_groups" in "device_data" -> "user_consent_records" -> "appID" section
+-- and_ SDL received SDL.OnAppPermissionsConsent (externalConsentStatus)
+-- SDL must
+-- add "consent_groups" to "device_data" -> "user_consent_records" -> "appID" section in Policy Table
+-- change_ according to the received externalConsentStatus settings in both:
+-- "consent_groups" in "device_data" -> "user_consent_records" -> "appID" section
+-- and
+-- "external_consent_status_groups" in "device_data" -> "user_consent_records" -> "appID" section
+--------------------------------------------------------------------------
+-- Test 07.01:
+-- Description: "consent_groups" does not exist. disallowed_by_external_consent_entities_on/off. HMI -> SDL: OnAppPermissionConsent(externalConsentStatus OFF)
+-- Expected Result: "consent_groups" is created with same status as externalConsentStatus
+--------------------------------------------------------------------------
+-- Precondition:
+-- Prepare JSON file with consent groups. Add all consent group names into app_polices of applications
+-- Request Policy Table Update.
+--------------------------------------------------------------------------
+Test["TEST_NAME_OFF".."_Precondition_Update_Policy_Table"] = function(self)
```

Comment:
@ttdong I kept the initial formatting of names, just added extra "" as otherwise it was not possible to process. 
Please note I am just making the script runnable, not updating it according to approved template.